src/Controller/IdeaController.php

```diff
@@ -152,6 +155,7 @@ public function show(
             'idea' => $idea,
             'totalSupporters' => $totalSupporters,
             'isMember' => $isMember,
+            'daysRemaining' => $daysRemaining,
```

Comment:
As you already pass the idea to twig, you don't need to pass your getRemainingDays, you can call it directly from twig